node/bin/src/prover_api/fri_job_manager.rs

```diff
@@ -88,6 +88,20 @@ impl FriJobManager {
         }
     }
 
+    /// Peek a batch data for a given batch number
+    pub fn peek_batch_data(&self, batch_number: u64) -> Option<ProverInput> {
+        match self.assigned_jobs.get_batch_data(batch_number) {
+            Some(prover_input) => {
+                tracing::info!("Batch data is peeked for batch number {batch_number}");
+                Some(prover_input)
+            }
+            None => {
+                tracing::debug!("No batch data found for batch number {batch_number}");
```

Comment:
this log will be confusing - it will just say "No batch data found for batch number {batch_number}" but there will be no context on why. If you want to log it, do "Trying to peel batch number {batch_number}  that is not present in `assigned_jobs`"